docs/api-reference/cli.md

```diff
@@ -138,7 +138,7 @@ optional arguments:
 ```
 usage: quilt3 push --dir DIR [-h] [--registry REGISTRY] [--dest DEST]
                    [--message MESSAGE] [--meta META] [--workflow WORKFLOW]
-                   [--force] [--dedupe] [--no-copy]
+                   [--force] [--dedupe] [--no-copy] [--detailed_help]
```

Comment:
Why not just make `-h` work like this instead of adding another command?

Reply:
`detailed_help` is the existing mechanism for adding documentation to the published web docs.